### DISCUSSION TITLE
Add Fine-tune for long memory conversations.

### DISCUSSION THREAD

**Discussion by slarrauri** [ChatGPTNextWeb/NextChat#911] [Ideas]
Basically, the idea is to use a pre-trained language model from OpenAI to generate responses to user questions in a chat, and then add the option to fine-tune the model with information from the current conversation. Then, apply iterative fine-tuning over the previously fine-tuned model.

**Comment by Yidadaa** [comment]
OpenAI currently does not provide access to fine-tuning ChatGPT models, so we can't do it.

**Reply by slarrauri** [reply]
OH, I see, Thanks.

**Comment by prakhar625** [comment]
They have recently launched fine tuning: https://platform.openai.com/docs/api-reference/fine-tuning
